src/components/DropzoneAreaBase.md

```diff
@@ -9,7 +9,8 @@ import { DropzoneAreaBase } from 'material-ui-dropzone';
 ```jsx
 <DropzoneAreaBase
   onAdd={(fileObjs) => console.log('Added Files:', fileObjs)}
-  onDelete={(fileObj) => console.log('Removed File:', fileObj)}
+  onDelete={(fileObj) => console.log('Removed File:', fileObj)}  
+  onAlert={(message, variant) => console.log(`${variant}: ${message}`)}
```

Comment:
Personally I don't think this makes sense for the "Basic Usage" however this is how it was originally in the thing Im porting, Perhaps once we have storybook we can sort out all the examples that are *really* just for the dev testing it works